code/datums/components/crafting/recipes.dm

```diff
@@ -475,15 +475,28 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_AMMO
 
-/datum/crafting_recipe/ishotgun
-	name = "Improvised Shotgun"
-	result = /obj/item/gun/ballistic/shotgun/doublebarrel/improvised
+/datum/crafting_recipe/musket
+	name = "Maintenance musket"
+	result = /obj/item/gun/ballistic/rifle/boltaction/musket
 	reqs = list(/obj/item/weaponcrafting/receiver = 1,
```

Comment:
PLEASE add a crafting recipe for receivers (maybe one involving a mousetrap?); currently, you basically have to print them from an autolathe, which kind of ruins the "ghetto gun constructed from maintenance materials" vibe of the improvised shotgun- er, maintenance musket

Reply:
Was thinking about this honestly. I'm hesitant to go tweaking this to be too constructable but yes the receiver is kind of a poor 'ghetto' constructed part.